app/components/sidebars/main/main_sidebar.test.js

```diff
@@ -82,4 +84,26 @@ describe('MainSidebar', () => {
         wrapper.setProps({theme: newTheme});
         expect(instance.render).toHaveBeenCalledTimes(1);
     });
+
+    test('should render main sidebar below PostList for iOS', () => {
+        Platform.OS = 'ios';
+
+        const wrapper = shallow(
+            <MainSidebar {...baseProps}/>
+        );
+        const drawer = wrapper.dive().childAt(1);
+        const drawerStyle = _.extend({}, ...drawer.props().style);
```

Comment:
```suggestion
        const drawerStyle = {...drawer.props().style};
```

Reply:
Hey @enahum - I went back and checked this. The data structure looks like this:
```
[ { position: 'absolute', top: 0, bottom: 0, zIndex: 3 },
      { backgroundColor: undefined, width: 710, left: 0, right: null },
      { transform: [ [Object] ] } ]
```
so I need to either use extend or reduce the objects into another object. Would you prefer I use `reduce` to avoid the `underscore`?
It'd look like this:
```
const drawerStyle = drawer.props().style.reduce((acc, obj) => ({...acc, ...obj}))
```

Reply:
went ahead and used reduce, it was simple enough anyways :) lmk if you'd like me to update it back.